<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="T3_CheckEI_Gen_Letter_Return_but">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return CheckEI Page</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T3_CheckEI_Gen_Letter_Execute_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>591</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>My_Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Senior_Initial_le"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>My_Senior_Initial</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_STO_Post_le"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>TO_Post</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>My_Initial_s</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Initial_le"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>STO_Post</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>My_Inital</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_TO_Post_le"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Initial_s_le"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Name_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>My Fax</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Fax_le"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Email_le"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Our_File_Ref"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>My Phone</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Date_le"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Our_File_Ref</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>My_Email</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_My_Phone_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>591</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Address_2_le"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Own_Consult_Address_3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Main_Contact_Name_le"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Fax_le"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Main_Contact_Post"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Own_Consult_Address_2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Own_Consult_Main_Contact_Post</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Own_Consult_Address_1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Address_1_le"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Address_3_le"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Address_4_le"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Own_Consult_Main_Contact_Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Fax</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Own_Consult_Address_4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Incoming_File_Ref_le">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Incoming_File_Ref</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Recur_FirstRound</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Own_Consult_Agreement_Title 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Own_Consult_Agreement_Title 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Consult_Agreement_Title_1_le"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Consult_Agreement_Title_2_le"/>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="T3_CheckEI_Gen_Letter_First_Round_rad">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>83</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>First Round</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="T3_CheckEI_Gen_Letter_Recurrent_rad">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>83</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Recurrent</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Batch_Nos"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Batch_Nos</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Rev_No_le"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_Own_Consult_Agreement_No"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Own_Consult_Agreement_No</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Rev_No</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="T3_CheckEI_Gen_Letter_No_of_Feature_le"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>No_of_Feature</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="2">
     <widget class="QLineEdit" name="T3_Letter_Docu_Title_le"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Docu_Title</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="T3_Letter_Reload_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-load</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>T3_CheckEI_Gen_Letter_My_Name_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_STO_Post_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_TO_Post_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_My_Initial_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_My_Senior_Initial_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_My_Initial_s_le</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_Execute_but</tabstop>
  <tabstop>T3_CheckEI_Gen_Letter_Return_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
